AUTHORS.txt

```diff
@@ -1,7 +1,12 @@
-# git log --format='%aN <%aE>' | sort -u
-
-Dmitry Mukhin <dm@uploadcare.com>
-Dmitry Petrov <dimaninc@gmail.com>
-Elijah <ilya@uploadcare.com>
-Gray Hound <admin@devilmaydie.name>
-Zarema Khalilova <zarema@uploadcare.com>
+# This list of authors is generated automatically based on the git history of this repo.
+
+Aleksandr Grenishin <nd0ut.me@gmail.com>
+Dmitry Mukhin <dm+github@uploadcare.com>
+Dmitry Mukhin <dm@uploadcare.com>
+Dmitry Petrov <dimaninc@gmail.com>
+Elijah <ilya@uploadcare.com>
+Gray Hound <admin@devilmaydie.name>
+Zarema <Zmoki@users.noreply.github.com>
+Zarema <zarema.khalilova@gmail.com>
+Zarema Khalilova <zarema.khalilova@gmail.com>
+Александр Гренишин <nd0ut.me@gmail.com>
```

Comment:
Since Alex Grenishin is still on the list of authors, I doubt we need the Cyrillic version of his name here.

Reply:
this list is autogenerated